<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_database</class>
 <widget class="QMainWindow" name="form_database">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>278</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>278</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Database</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../gui/ControlVersions/Hos3.9/icon/database-storage.png</normaloff>../../../gui/ControlVersions/Hos3.9/icon/database-storage.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Connect Database</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineEdit_server">
         <property name="placeholderText">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_dbname">
         <property name="placeholderText">
          <string>Database Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="lineEdit_port">
         <property name="placeholderText">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="lineEdit_user">
         <property name="placeholderText">
          <string>User</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionVersion_Postgres">
   <property name="text">
    <string>Version Postgres</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
